tun/src/unix/linux/mod.rs

```diff
@@ -68,6 +67,17 @@ impl TunInterface {
         self.perform(|fd| unsafe { sys::if_set_addr(fd, &iff) })?;
     }
 
+    #[throws]
+    pub fn set_iface_up(&self) {
+        std::process::Command::new("ip")
+            .arg("link")
+            .arg("set")
+            .arg("dev")
+            .arg(&self.name().unwrap())
+            .arg("up")
+            .output()?;
+    }
+
```

Comment:
We should not be calling other commands directly.

Reply:
I'm aware it's not ideal.
I lost 3 hours to this reading linux header files; `/dev/net/tun` just doesen't want to listen to me when I set flags. It's quite a travesty actually.

Reply:
https://docs.rs/rtnetlink/latest/rtnetlink/struct.LinkSetRequest.html#method.up

Look at using this method instead

Reply:
It's definitely only a temporary solution.